### DISCUSSION TITLE
github dan nasil puan kazaniriz

### DISCUSSION THREAD

**Discussion by mlkmutlu** [IBAX-io/go-ibax#1868] [Bounty Hunter]
github dan nasil puan kazaniriz.burada yapmamiz gereken gorevler nelerdir.bilen yokmu

**Comment by Mslmksc27** [comment]
Bende hala anlamaya çalışıyorum.9 Şub 2022 17:41 tarihinde mlkmutlu ***@***.***> yazdı:
github dan nasil puan kazaniriz.burada yapmamiz gereken gorevler nelerdir.bilen yokmu
